src/backend/templates/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-45266465-2"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+      gtag('config', 'UA-45266465-2');
+    </script>
+    <meta charset="utf-8">
+    <title>The Cycle-GAN App</title>
+    <link rel= "stylesheet" type= "text/css" href= "{{ url_for('static',filename='styles/style.css') }}">
+    <link rel="shortcut icon" href="{{ url_for('static', filename='favicon.ico') }}">
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
+  <body>
+    <header>
+      <h1>The Cycle-GAN App</h1>
+    </header>
+    <div>
+      <left>
+        <section>
+          <img id='uploaded_image' height='256' width='256'>
+        </section>
+        <form id='upload_form' action='' enctype='multipart/form-data' method='post'>
```

Comment:
Can we remove 'action='''?

Reply:
good point, we can actually remove everything and only leave '<form>' because these are artifacts from when I used the form.submit functionality